src/src/ManagerSync.php

```diff
@@ -92,18 +92,18 @@ public function enforce_latest_version(): void {
 		}
 
 		$latest_version = $this->environment->get_cache()->get_manager_sync_data( 'latest_cli_version' );
-		$enforce_latest = $this->environment->get_cache()->get_manager_sync_data( 'enforce_latest' );
+		$minimum_cli_version = $this->environment->get_cache()->get_manager_sync_data( 'minimum_cli_version' );
 
 		if ( version_compare( $current_version, $latest_version, '<' ) ) {
-			if ( $enforce_latest ) {
+			$this->output->writeln( '<comment>There\'s a new version of the QIT CLI available!</comment>' );
+			$this->output->writeln( sprintf( "Current version: %s\n<info>Latest version: %s</info>\nIt's strongly recommended to update. Update instructions: https://woocommerce.github.io/qit-documentation/#/cli/getting-started?id=updating-qit</info>\n", $current_version, $latest_version ) );
```

Comment:
I'm debating a bit on the wording here, could we upsell a bit more, something like:

```suggestion
			$this->output->writeln( sprintf( "Current version: %s\n<info>Latest version: %s</info>\nUpdate today to take advantage of the latest features! Update instructions: https://woocommerce.github.io/qit-documentation/#/cli/getting-started?id=updating-qit</info>\n", $current_version, $latest_version ) );
```